spec/index.bs

```diff
@@ -11194,6 +11452,30 @@ It must only be included by interfaces which also include those mixins.
                         - [$effective buffer binding size$](|bound|) must be &ge; [=minimum buffer binding size=]
                             of the binding variable in |pipeline|'s shader that corresponds to |bindGroupEntry|.
             - [$Encoder bind groups alias a writable resource$](|encoder|, |pipeline|) must be `false`.
+
+            <div class=compatmode>
+                - If |encoder|.{{GPUObjectBase/[[device]]}}.{{device/[[features]]}} does not [=list/contain=]
+                    {{GPUFeatureName/"core-features-and-limits"}}:
+                    - All bindings referring to the same {{GPUTexture}} must have compatible {{GPUTextureView}}s:
+
+                        For each pair of ({{GPUIndex32}} |index1|, {{GPUBindGroupLayout}} |bindGroupLayout1|) in |bindGroupLayouts|:
+                        - If |bindGroupLayout1| is `null`, [=iteration/continue=].
+                        - Let |bindGroup1| be |encoder|.{{GPUBindingCommandsMixin/[[bind_groups]]}}[|index1|].
+                        - For each {{GPUBindGroupEntry}} |bindGroupEntry1| in |bindGroup1|.{{GPUBindGroup/[[entries]]}}:
+                            - If |bindGroupEntry1|.{{GPUBindGroupEntry/resource}} is not a {{GPUTextureView}}, [=iteration/continue=].
+                            - Let |descriptor1| be |bindGroupEntry1|.{{GPUBindGroupEntry/resource}}.{{GPUTextureView/[[descriptor]]}}.
+                            - For each pair of ({{GPUIndex32}} |index2|, {{GPUBindGroupLayout}} |bindGroupLayout2|) in |bindGroupLayouts|:
+                                - If |bindGroupLayout2| is `null`, [=iteration/continue=].
+                                - Let |bindGroup2| be |encoder|.{{GPUBindingCommandsMixin/[[bind_groups]]}}[|index2|].
+                                - For each {{GPUBindGroupEntry}} |bindGroupEntry2| in |bindGroup2|.{{GPUBindGroup/[[entries]]}}:
+                                    - If |bindGroupEntry2|.{{GPUBindGroupEntry/resource}} is not a {{GPUTextureView}}, [=iteration/continue=].
+                                    - If |bindGroupEntry1|.{{GPUBindGroupEntry/resource}} is not equal to |bindGroupEntry2|.{{GPUBindGroupEntry/resource}}, [=iteration/continue=].
```

Comment:
I think there is a bug here. Was it meant to be "is equal to" to ignore trivially-equal texture views?
If so that's not really necessary, I'd just skip it.
```suggestion
```

Reply:
Nvm it wasn't that. Fix in #5415